dev/Materials/Acrylic/AcrylicTestApi.idl

```diff
@@ -20,7 +20,7 @@ runtimeclass AcrylicTestApi : Windows.UI.Xaml.DependencyObject
 interface IAcrylicBrushStaticsPrivate
 {
     Windows.UI.Composition.CompositionEffectBrush CreateBackdropAcrylicEffectBrush(Windows.UI.Composition.Compositor compositor, Windows.UI.Color initialTintColor, Windows.UI.Color initialFallbackColor, Boolean willTintColorAlwaysBeOpaque);
-    Windows.UI.Composition.CompositionEffectBrush CreateBackdropAcrylicEffectBrushWithLuminosity(Windows.UI.Composition.Compositor compositor, Windows.UI.Color initialTintColor, Windows.UI.Color initialLuminosityColor, Windows.UI.Color initialFallbackColor, Boolean willTintColorAlwaysBeOpaque);
+    Windows.UI.Composition.CompositionEffectBrush CreateBackdropAcrylicEffectBrushWithLuminosity(Windows.UI.Composition.Compositor compositor, Windows.UI.Color initialTintColor, Windows.Foundation.IReference<double> luminosityOpacity, Windows.UI.Color initialFallbackColor, Boolean willTintColorAlwaysBeOpaque);
```

Comment:
I assume this breaking change is OK because only Shell uses these private APIs, and Shell will move to the new one as part of consuming WinUI 2.x version here.